spec/intl.html

```diff
@@ -1791,6 +1772,30 @@ <h1>
             In a _calendar_ such as *"japanese"* where eras do not start and end at year and/or month boundaries, note that the returned List should contain *"era"* and *"eraYear"* if _keys_ contains *"day"*, *"month"*, or *"monthCode"* (not only if it contains *"era"*, *"eraYear"*, or *"year"*, as in the example above) because it's possible for changing the day or month to cause a conflict with the era.
```

Comment:
Another minor clarification that may be helpful while we're in the neighborhood:

```suggestion
            In a _calendar_ such as *"japanese"* where eras do not start and end at year and/or month boundaries, note that the returned List should contain *"era"* and *"eraYear"* if _keys_ contains *"day"*, *"month"*, or *"monthCode"* (not only if it contains *"era"*, *"eraYear"*, or *"year"*, as in the example above) because Japanese calendar eras can start in the middle of a year, which makes it possible that changing the day or month can cause a conflict with the era.
```

Reply:
Actually, ignore that suggestion: I skipped over the statement at the top of the paragraph that talked about eras starting midyear.